docs/guides/js-waku/use-waku-react.md

```diff
@@ -0,0 +1,356 @@
+
+```shell
+npm install @waku/react @waku/sdk
+```
+
+</TabItem>
+<TabItem value="yarn" label="Yarn">
+
+```shell
+yarn add @waku/react @waku/sdk
+```
+
+</TabItem>
+</Tabs>
+
+## Create a Relay Node
+
+Use the `useCreateRelayNode()` hook to create a Relay Node:
+
+```js title="App.js"
+import { useCreateRelayNode } from "@waku/react";
+
+function App() {
+	// Create and start a Relay Node
+	const { node, error, isLoading } = useCreateRelayNode({
+		options: {
+			defaultBootstrap: true,
+			emitSelf: true,
+		}
+	});
+
+	// "node" is the created Relay Node
+	// "error" captures any error that occurs during node creation
+	// "isLoading" indicates whether the node is still being created
+	// Use these to handle states and render the UI in your application
+}
+```
+
+## Create a Light Node
+
+Use the `useCreateLightNode()` hook to create a Light Node and specify the [protocols](/overview/concepts/protocols) for remote peers:
+
+```js title="App.js"
+import { useCreateLightNode } from "@waku/react";
+import { Protocols } from "@waku/interfaces";
+
+function App() {
+	// Create and start a Light Node and wait for remote peers
+	const { node, error, isLoading } = useCreateLightNode({
+		options: { defaultBootstrap: true },
+		protocols: [Protocols.LightPush, Protocols.Filter],
+	});
+
+	// "node" is the created Light Node
+	// "error" captures any error that occurs during node creation
+	// "isLoading" indicates whether the node is still being created
+	// Use these to handle states and render the UI in your application
+}
+```
+
+## Create an Encoder and Decoder
+
+Use the `useCreateContentPair()` hook to create a message `encoder` and `decoder` pair:
+
+```js title="App.js"
+import { useCreateContentPair } from "@waku/react";
+
+function App() {
+	// Choose a content topic
+	const contentTopic = "/waku-react-guide/1/message/utf8";
+	const ephemeral = false;
+
+	// Create a message encoder and decoder pair
+	const { encoder, decoder } = useCreateContentPair(contentTopic, ephemeral);
+
+	// "encoder" is the message encoder
+	// "decoder" is the message decoder
+	// Use these to handle the messages in your application
+}
+```
+
+## Send Messages Using Light Push
+
+Use the `useLightPush()` hook to bind `Light Push` methods to a node and `encoder`:
+
+```js title="App.js"
+import {
+	useCreateLightNode,
+	useCreateContentPair,
+	useLightPush,
+} from "@waku/react";
+import { Protocols } from "@waku/interfaces";
+import { utf8ToBytes } from "@waku/sdk";
+
+function App() {
+	// Create and start a Light Node and wait for remote peers
+	const { node, error, isLoading } = useCreateLightNode({
+		options: { defaultBootstrap: true },
+		protocols: [Protocols.LightPush, Protocols.Filter],
+	});
+
+	// Choose a content topic and create an encoder
+	const contentTopic = "/waku-react-guide/1/message/utf8";
+	const { encoder  } = useCreateContentPair(contentTopic);
+
+	// Wait for the node to finish loading before sending messages
+	// (isLoading === false)
```

Comment:
How would the code look like?

I think this example needs to be a bit more verbose. Where you actually define a button to send messages and the button is `disabled` whne `isLoading` is true.

Reply:
it can be as simple as 

```jsx
if (isLoading) {
  return <Loader />
}
```

or something